### PR TITLE
rate limit: test inbound service account local rate limit

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: tests/integration/telemetry/policy/envoy_ratelimit_test.go [hunk 1]
```diff
@@ -79,6 +79,17 @@ func TestLocalRouteSpecificRateLimiting(t *testing.T) {
 		})
 }
 
+func TestLocalRateLimitingServiceAccount(t *testing.T) {
+	framework.
+		NewTest(t).
+		Features("traffic.ratelimit.envoy").
+		Run(func(ctx framework.TestContext) {
+			setupEnvoyFilter(ctx, "testdata/enable_envoy_local_ratelimit_sa.yaml")
```
Comment (bianpengyuan) [inline tests/integration/telemetry/policy/envoy_ratelimit_test.go:87]: Probably clean up the EnvoyFilter after test finishes? Looking at other test in this package, and seems like they all do not do clean up, and I cannot find clean up is performed anywhere.

Comment (kyessenov) [reply tests/integration/telemetry/policy/envoy_ratelimit_test.go]: yeah, this is pretty bad. added a cleanup.

**Review by bianpengyuan — APPROVED** [review]

**Comment by kyessenov** [comment]
/retest
Unrelated failures.

**Pull request merged**